src/aat/resources/features/F-052/F-052.feature

```diff
@@ -0,0 +1,49 @@
+@F-052
+Feature: F-052: Get case data for a given case type for Citizen
```

Comment:
I took the feature wording from the latest excel file, so I am keeping this as is, otherwise it would cause issue to find the feature related folder. Let me know if you still want me to update that, I am happy to do that.